cocos/audio/audio-manager.ts

```diff
@@ -97,6 +97,7 @@ export class AudioManager {
             });
         }
         if (audioInfoToDiscard) {
+            // eslint-disable-next-line @typescript-eslint/no-floating-promises
             audioInfoToDiscard.audio.stop();
```

Comment:
```suggestion
            audioInfoToDiscard.audio.stop().catch((err) => {});
```

it‘s better to catch the error of promise

Reply:
fixed